docs/core/getting_started/installation.md

```diff
@@ -54,3 +54,7 @@ The extra packages include:
 ## Development
 
 For developing Prefect, see the [development guide](../development/overview.md).
+
+## Docker
+
+Prefect provides Docker images for master builds and versioned releases [here](https://hub.docker.com/r/prefecthq/prefect).
```

Comment:
I think we should describe our tagging policy in at least some detail (e.g., what types of tags, what do they mean, etc.)

Reply:
They're on the readme of the Dockerhub page. Do you think it should also go here?

Reply:
Oh nice -- but yea I do actually think we should copy / paste that exact table here.  I didn't even know that table was there because it's at the bottom of the standard README